assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -606,7 +606,9 @@ che.server.secure_exposer.jwtproxy.token.issuer=wsmaster
 che.server.secure_exposer.jwtproxy.token.ttl=8800h
 che.server.secure_exposer.jwtproxy.auth.loader.path=/_app/loader.html
 che.server.secure_exposer.jwtproxy.image=quay.io/eclipse/che-jwtproxy:0.10.0
+che.server.secure_exposer.jwtproxy.memory_request=10mb
```

Comment:
LGTM, but taking a look at the metrics at OSD v4 maybe increasing request to 15mb to be on the safe side

![image](https://user-images.githubusercontent.com/1461122/101615565-904f4780-3a0e-11eb-8613-a6c0c9f55c9a.png)

Reply:
updated